config/commands.js

```diff
@@ -385,7 +385,7 @@ var commands = exports.commands = {
 				if (move.flags['sound']) details["<font color=black>&#10003; Sound</font>"] = "";
 				if (move.flags['bullet']) details["<font color=black>&#10003; Bullet</font>"] = "";
 				if (move.flags['pulse']) details["<font color=black>&#10003; Pulse</font>"] = "";
-				if (move.flags['protect']) details["<font color=black>&#10003; Blocked by Protect</font>"] = "";
+				if (!move.flags['protect'] && !/(ally|self)/i.test(move.target)) details["<font color=black>&#10003; Ignores Protect</font>"] = "";
```

Comment:
`/ally|self|side|all/i`

"side" and "all" are there because you don't want Stealth Rocks (`target: "foeSide"`) or Rain Dance (`target: "all"`) to come up as "not blocked by Protect" either.